docs/api/manual/UseCases.md

```diff
@@ -308,97 +290,97 @@ After the receiver is done with the message, it must free resources backing the
 
-By naively reimplementing the traditional sort of secure gateway in terms of uVisor IPC, we would lose out on some of the static verifiability promises of the original uVisor.
+By naively reimplementing the traditional sort of secure gateway in terms of uVisor IPC, you lose some of the static verifiability promises of the original uVisor.
 
 Previously, a secure gateway stored the following information in flash.
-  - uVisor Secure Gateway Magic (to identify a potential secure gateway)
-  - Target Box (Config Pointer)
-  - Target Function
+  - uVisor Secure Gateway Magic (to identify a potential secure gateway).
+  - Target Box (Config Pointer).
+  - Target Function.
 
 To ensure that such static verifiabiillty promises are maintained, the secure gateway is enhanced to include the following information in flash.
-  - IPC uVisor Secure Gateway Magic (to identify a potential IPC secure gateway)
-  - Source Box (Config Pointer)
-  - Target Box Queue Name
-  - Target Box Namespace (optional)
+  - IPC uVisor Secure Gateway Magic (to identify a potential IPC secure gateway).
+  - Source Box (Config Pointer).
+  - Target Box Queue Name.
+  - Target Box Namespace (optional).
 
-Secure gateway calls are synchronous, to avoid changing semantics. When the target process completes the function call on behalf of the caller, the source process receives the return code for the function call. Complex types (such as a struct) cannot be returned via a secure gateway call.
+Secure gateway calls are synchronous, to avoid changing semantics. When the target process completes the function call on behalf of the caller, the source process receives the return code for the function call. Secure gateway calls cannot return complex types, such as structs.
 
 An asynchronous secure gateway is also introduced, to make asynchronous RPC easier than setting up a queue and using `uvisor_ipc_send`.
 -->
 
+## RTOS integration mechanics
 
-### RTOS Integration Mechanics
+As currently integrated with uVisor, the RTOS runs with uVisor privileges. This must be fixed. [This issue is tracked on GitHub](https://github.com/ARMmbed/uvisor/issues/235). For now, the RTOS is a privileged component of the overall system, sharing privileged residence with uVisor. uVisor and the RTOS must trust each other because of this. Generally, uVisor is the manager of interrupts and the MPU, and the RTOS is the manager of context switching.
 
```

Comment:
This paragraph is confusing to me. Is it saying that RTOS has privileges but shouldn't? If so, why does the next paragraph list capabilities that support such integration? If not, what is it saying?